cli/jobs/nebulaml/bert-pretrain/src/train.py

```diff
@@ -0,0 +1,130 @@
+from datasets import *
+from transformers import *
+from tokenizers import *
+import os
+import json
+
+
+if __name__ == "__main__":
+
+    parser = HfArgumentParser(TrainingArguments)
+    training_args, args = parser.parse_args_into_dataclasses(return_remaining_strings=True)
+
+    #bookcorpus = load_dataset("bookcorpus", split="train")
+    #wikide = load_dataset("wikipedia", "20220301.de", split="train")
```

Comment:
nit: Remove the commented calls if not required.

Reply:
removed.